src/views/network/view_session.cpp

```diff
@@ -107,9 +128,132 @@ namespace big
 
 		components::sub_title("All Players");
 		ImGui::Checkbox("Off The Radar", &g->session.off_radar_all);
+		ImGui::SameLine();
 		ImGui::Checkbox("Never Wanted", &g->session.never_wanted_all);
+		ImGui::SameLine();
 		ImGui::Checkbox("Semi Godmode", &g->session.semi_godmode_all);
 
+		ImGui::Checkbox("Explosion Karma", &g->session.explosion_karma);
+		ImGui::SameLine();
+		ImGui::Checkbox("Damage Karma", &g->session.damage_karma);
+
+		static int global_wanted_level = 0;
+
+		if (ImGui::SliderInt("Wanted Level", &global_wanted_level, 0, 5))
+		{
+			*scr_globals::globalplayer_bd.at(self::id, scr_globals::size::globalplayer_bd).at(213).as<int*>() = global_wanted_level;
+		}
+
+		ImGui::SameLine();
+		if (ImGui::Checkbox("Force", &g->session.wanted_level_all))
+		{
+			*scr_globals::globalplayer_bd.at(self::id, scr_globals::size::globalplayer_bd).at(212).as<Player*>() = __rdtsc() + 32;
+			*scr_globals::globalplayer_bd.at(self::id, scr_globals::size::globalplayer_bd).at(213).as<int*>() = global_wanted_level;
+		}
+
+		components::button("Kill Everyone", [] { g_player_service->iterate([](auto& plyr) { toxic::kill_player(plyr.second, g_player_service->get_self()); }); });
+
+		ImGui::SameLine();
+
+		components::button("Turn Everyone Into Beast", [] { toxic::turn_everyone_into_beast(); });
+		if (ImGui::IsItemHovered())
+			ImGui::SetTooltip("Including you");
+
+		components::button("Give All Weapons", [] { g_player_service->iterate([](auto& plyr) { toxic::give_all_weapons(plyr.second); script::get_current()->yield(450ms); }); });
+		ImGui::SameLine();
+		components::button("Remove All Weapons", [] { g_player_service->iterate([](auto& plyr) { toxic::remove_all_weapons(plyr.second); }); });
+
+		components::button("CEO Kick", [] { 
+			g_player_service->iterate([](auto& plyr) 
+			{
+				if (*scr_globals::gpbd_fm_3.at(plyr.second->id(), scr_globals::size::gpbd_fm_3).at(10).as<int*>() != -1)
+					toxic::ceo_kick(plyr.second); 
+			}); 
+		});
+
+		components::button("CEO Ban", [] { 
+			g_player_service->iterate([](auto& plyr) 
+			{ 
+				if (*scr_globals::gpbd_fm_3.at(plyr.second->id(), scr_globals::size::gpbd_fm_3).at(10).as<int*>() != -1)
+					toxic::ceo_ban(plyr.second); 
+			}); 
+		});
+
+		components::small_text("Teleports");
+
+		static int selected_apartment_idx = 1;
+		static int selected_warehouse_idx = 1;
```

Comment:
Why not save this?
And maybe have global tp idx and player tp idx as 1 variable.